docs/site/BelongsTo-relation.md

```diff
@@ -43,6 +43,28 @@ related routes, you need to perform the following steps:
 This section describes how to define a `belongsTo` relation at the model level
 using the `@belongsTo` decorator to define the constraining foreign key.
 
+The definition of the `belongsTo` relation is inferred by using the `@belongsTo`
+decorator. The decorator takes in a function resolving the target model class
+constructor and designates the relation type. It also calls `property()` to
+ensure that the decorated property is correctly defined. Usage of
+belongsTo relation is similar to legacy [datasource juggler](https://github.com/strongloop/loopback-datasource-juggler).
+
+The `@belongsTo` decorator takes three parameters:
+
+- `target model class` (mandatory)
```

Comment:
Can you change `mandatory` to `required`? We usually use `required`.

Reply:
agree